jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxChecker.java

```diff
@@ -592,6 +594,12 @@ private void checkDataObject(DataObject dataObject) {
 		}
 	}
 
+	private void checkMandatoryDefaultGate(Gateway gateway, SequenceFlow defaultSequenceFlow) {
+		if(defaultSequenceFlow == null || (gateway.getOutgoing() != null && !gateway.getOutgoing().contains(defaultSequenceFlow))) {
+			addError(gateway, new ValidationSyntaxError(gateway, BPMN2_TYPE, SyntaxCheckerErrors.NOT_VALID_DEFAULT_GATE));
+		}
+	}
+
```

Comment:
Hello @tsurdilo. I agree, I will amend some changes into #248 and then I will close this PR. Thanks.